player/lua/stats.lua

```diff
@@ -1431,12 +1431,12 @@ cache_recorder_timer:kill()
 -- Current page and <page key>:<page function> mapping
 curr_page = o.key_page_1
 pages = {
-    [o.key_page_1] = { f = default_stats, desc = "Default" },
-    [o.key_page_2] = { f = vo_stats, desc = "Extended Frame Timings", scroll = true },
-    [o.key_page_3] = { f = cache_stats, desc = "Cache Statistics" },
-    [o.key_page_4] = { f = keybinding_info, desc = "Active Key Bindings", scroll = true },
-    [o.key_page_5] = { f = track_info, desc = "Selected Tracks Info", scroll = true },
-    [o.key_page_0] = { f = perf_stats, desc = "Internal Performance Info", scroll = true },
+    [o.key_page_1] = { idx = "1", f = default_stats, desc = "Default" },
+    [o.key_page_2] = { idx = "2", f = vo_stats, desc = "Extended Frame Timings", scroll = true },
+    [o.key_page_3] = { idx = "3", f = cache_stats, desc = "Cache Statistics" },
+    [o.key_page_4] = { idx = "4", f = keybinding_info, desc = "Active Key Bindings", scroll = true },
+    [o.key_page_5] = { idx = "5", f = track_info, desc = "Selected Tracks Info", scroll = true },
+    [o.key_page_0] = { idx = "6", f = perf_stats, desc = "Internal Performance Info", scroll = true },
```

Comment:
This page should be `0`
```suggestion
    [o.key_page_0] = { idx = "0", f = perf_stats, desc = "Internal Performance Info", scroll = true },
```